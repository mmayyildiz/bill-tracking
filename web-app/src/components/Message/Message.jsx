import React from 'react';
import './style.css';

const Message = ({msg}) => {
    return (
        <div>
          <h3 className="message">{msg}</h3>
        </div>
      );
}

export default Message;