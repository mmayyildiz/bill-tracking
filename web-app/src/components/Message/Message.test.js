import React from 'react';
import { shallow } from 'enzyme';
import Message from './Message';

const props = {
    msg: 'dummy'
 }

 const mountedComponent = shallow(<Message {...props}/>);

describe('Message', () => {

    it('should render with props', () => {
        expect(mountedComponent).toMatchSnapshot();
    });

    it('should render a message', () => {
        expect(mountedComponent.find('h3').html()).toContain('dummy');
    });

});
