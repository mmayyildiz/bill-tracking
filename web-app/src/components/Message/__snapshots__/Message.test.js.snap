// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message should render with props 1`] = `ShallowWrapper {}`;
